src/parser.rs

```diff
@@ -250,6 +250,18 @@ pub fn initialize() -> App<'static, 'static> {
                     "Specify HTTP headers (ex: -H Header:val 'stuff: things')",
                 ),
         )
+        .arg(
+            Arg::with_name("cookies")
+                .short("b")
+                .long("cookies")
+                .value_name("COOKIE")
+                .takes_value(true)
+                .multiple(true)
```

Comment:
multiple values are allowed in the parser definition, but only a single `Cookie: stuff=things` makes it into the actual request (the last one to be iterated over in the loop).

A single Cookie is preferred, as it's HTTP/1.1 compliant, so let's remove the `multiple(true)` and make sure documentation shows how to send multiple key/values in a single `--cookies` param.